src/common/models/User.ts

```diff
@@ -60,6 +60,8 @@ export type ComputedUserProperties = {
   _profile_picture_color?: Color
   _artist_pick?: ID
   _has_reposted?: boolean
+  updatedProfilePicture: any
```

Comment:
Sorry for review delay. This can be
`{ file: File, url: string } | null`

Reply:
or rather `updateProfilePicture?: { file: File, url: string }`

Reply:
the `?` is necessary for the ts compile to work (see CI)

Reply:
indeed, i rushed the typing there